lib/cradle.js

```diff
@@ -135,20 +139,20 @@ cradle.Connection = function Connection(/* variable args */) {
 cradle.Connection.prototype.rawRequest = function (options, callback) {
     var promise = new(events.EventEmitter), 
         self = this;
-
+    
     // HTTP Headers
     options.headers = options.headers || {};
 
-    // Set HTTP Basic Auth
-    if (this.auth) {
```

Comment:
It might not be the best _time_ to remove that feature, nor may it be the best _way_ right now, but the way configuration is done now prevents users from using the full _force_ of the underlying request library (which is pretty vital).

I should redo this pull request to be more *backwards* compatible, but the issue with this particular line of code is that cradle is re-making functionality that already exists in the `request` library (and with more features).

Cradle is a really great library, but if I can't get through hurdles placed on me by proxies because I can't configure `request` correctly that is a real good reason for me to use something else. That being said, I'm going to go do this in a more backwards compatible way and get back to you.